HW5/task_1_loop.py

```diff
@@ -0,0 +1,17 @@
+# ===============================================  TASK 1 =============================================
+
+text = input("Enter something pls: ")
+
+for word in text:
```

Comment:
nit: I think it would be better to name it a letter or character or symbol because actually, it iterates by not a word